reo/validators.py

```diff
@@ -814,6 +815,38 @@ def remove_nones(self, object_name_path, template_values=None, real_values=None,
                         if input_isDict == False:
                             self.input_as_none.append([name, object_name_path[-1] + ' (number {})'.format(number)])
 
+    
+    def check_constraint_ranges(self, object_name_path, template_values=None, real_values=None, number=1, input_isDict=None):
```

Comment:
What about naming this `check_min_less_than_max` ? And perhaps we should move other checks for min < max to this method? (I know that we have one for outage_start/end_time_step in the validator).